Adaption of functional comparision in Quality Guidelines for Apps and Themes based on App System (#920)
shopware/docs: resources/guidelines/testing/store/quality-guidelines-apps/README.md

> **Changelog**
>

> >> 12/04/23: We are now checking for a functional comparison with functions from our Rise or above edition. 

Reviewer: ```suggestion
>> 12/04/23: Check for a functional comparison with functions from the Rise or above edition.
```

* We check the complete functionality of the app (including the uninstallation and reinstallation procedure).

> * We want to improve the quality of the Shopware Community Store and offer as many different apps as possible. Hence, we check for a functional comparison with other apps already present in the Shopware Community Store or in our Rise or above edition. If there is an app with the same function, it can be rejected as it doesn't provide any added value. For further information, write an email to [alliances@shopware.com](mailto:alliances@shopware.com).

Reviewer: ```suggestion
* We want to improve the quality of the Shopware Community Store and offer as many different apps as possible. Hence, we check for a functional comparison with other apps already in the Shopware Community store, in the Rise edition, or above. If there is an app with the same function, it can be rejected as it doesn't provide any added value. For further information, write an email to [alliances@shopware.com](mailto:alliances@shopware.com).
```
